### PR TITLE
use ks setting

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**
- use ks-setting 
- can not remove token list, it currently uses in some places in project.

### CODE REVIEW

File: src/components/TokenLogo/index.js [hunk 1]
```diff
@@ -1,18 +1,11 @@
...
-import { getMumbaiTokenLogoURL } from '../../utils/mumbaiTokenMapping'
-import { getBscTestnetTokenLogoURL } from '../../utils/bscTestnetTokenMapping'
-import { getBscTokenLogoURL } from '../../utils/bscTokenMapping'
-import { getAvaxTokenLogoURL } from '../../utils/avaxTokenMapping'
-import { getFantomTokenLogoURL } from '../../utils/fantomTokenMapping'
-import { getCronosTokenLogoURL } from '../../utils/cronosTokenMapping'
-import { ChainId } from '../../constants/networks'
-import { useTokensList } from '../../contexts/NetworkInfo'
+
+import { useGlobalData, useWhitelistTokensMap } from '../../contexts/GlobalData'
```
Comment (hungdoansy) [inline src/components/TokenLogo/index.js:7]: `useGlobalData` can be removed. not in use

File: .eslintrc.js [hunk 2]
```diff
@@ -20,6 +20,8 @@ module.exports = {
     'no-unused-vars': 0,
     'no-extra-boolean-cast': 0,
     'no-inner-declarations': 0,
+    'no-constant-condition': 0,
```
Comment (hungdoansy) [inline .eslintrc.js:23]: I think this rule can be disabled one time. No need to disable this for the whole project

File: .eslintrc.js [hunk 3]
```diff
@@ -20,6 +20,8 @@ module.exports = {
...
+    'no-async-promise-executor': 0,
```
Comment (hungdoansy) [inline .eslintrc.js:24]: Maybe remove this line? I don't see anywhere violates this rule

Comment (nguyenhoaidanh) [reply .eslintrc.js:24]: it catch eslint error when
white(true)
vs
new Promise(async(resolve,reject)=>{...})

I see it too stupid :D but ok, I will remain it for u.

Comment (hungdoansy) [reply .eslintrc.js:24]: Line 23 is needed, but line 24 is not, i don't see async promise in your changes

Comment (nguyenhoaidanh) [reply .eslintrc.js]: updated, continue to review 2 project please

**Review by hungdoansy — APPROVED** [review]

**Review by namgold — APPROVED** [review]

File: src/components/TokenLogo/index.js [hunk 4]
```diff
@@ -48,28 +41,18 @@ const StyledHelpCircle = styled(HelpCircle)`
...
-  )
-}
-
 export default function TokenLogo({ address, networkInfo, header = false, size = '24px', ...rest }) {
   const [error, setError] = useState(false)
-  const tokensList = useTokensList()?.[networkInfo.chainId]
 
+  const { chainId } = networkInfo
+
+  const tokensList = useTokensList()?.[chainId]
```
Comment (namgold) [inline src/components/TokenLogo/index.js:49]: It would be better to move `[chainId]` into `useTokenlist()` to reduce re-render

**Pull request merged**